demo_parallel/src/main/java/com/novoda/downloadmanager/demo/parallel/DemoParallelApplication.java

```diff
@@ -0,0 +1,12 @@
+package com.novoda.downloadmanager.demo.parallel;
+
+import android.app.Application;
+
+import com.novoda.downloadmanager.lib.DownloadClientReadyChecker;
+
+public class DemoParallelApplication extends Application implements DownloadClientReadyChecker {
```

Comment:
In the readme we explicitly wrote that the example for custom rules was in the serial demo not the parralel. No need to add this